parm/config.wcoss2.yaml

```diff
@@ -0,0 +1,90 @@
+obsforge:
+  PSLOT: obsforge
+  HOMEobsforge: /lfs/h2/emc/da/noscrub/cory.r.martin/obsforge/obsforge/ 
+  SDATE: 202504250000
+  EDATE: 202506250000
+  COMROOT: /lfs/h2/emc/ptmp/cory.r.martin/obsforge
+  DCOMROOT: /lfs/h1/ops/prod/dcom/ 
+  DATAROOT: /lfs/h2/emc/stmp/cory.r.martin/RUNDIRS
+  SCHEDULER: pbspro
+  ACCOUNT: GFS-DEV
+  QUEUE: dev
+  PARTITION: dev 
+  KEEPDATA: NO
```

Comment:
Consider running with **KEEPDATA: YES** before code handoff to NCO (current practice)